jina/helper.py

```diff
@@ -949,7 +949,7 @@ def typename(obj):
     if not isinstance(obj, type):
         obj = obj.__class__
     try:
-        return f'{obj.__module__}.{obj.__name__}'
```

Comment:
I think typename should not be touched. If the name is taken from the `typename` is some downstream problem. But typename may be better if it returns all.

If the code is only used to identify the executor, then it would need to change its name this method

Reply:
Do you like this better? https://github.com/jina-ai/jina/pull/2486/commits/c8ccfd7706b3c6aa3d15a52b428e5e87347c98ea